interop-java/src/main/java/scalaz/zio/java/data/Tuple.java

```diff
@@ -0,0 +1,30 @@
+package scalaz.zio.java.data;
+
+public final class Tuple<L, R> {
```

Comment:
Tuple2

Reply:
Renamed to Pair, since it's what Spring uses.

Reply:
How about make use of https://github.com/sbt/sbt-boilerplate to generate more tuples?